proto/greptime/v1/region/server.proto

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package greptime.v1.region;
+
+option java_package = "io.greptime.v1";
+option java_outer_classname = "Server";
+option go_package = "github.com/GreptimeTeam/greptime-proto/go/greptime/v1";
```

Comment:
Do we still need them(java/go)?

Reply:
At present only rust is needed. So remove other languages?

Reply:
I think we can remove other languages

Reply:
I find only go need to generate specifically, other are auto included. I've reverted the go script